openmetadata-ui/src/main/resources/ui/src/components/Webhooks/WebhooksV1.tsx

```diff
@@ -0,0 +1,183 @@
+/*
+ *  Copyright 2021 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { Card, Select } from 'antd';
+import classNames from 'classnames';
+import { isNil, startCase } from 'lodash';
+import React, { FC, useEffect, useState } from 'react';
+import {
+  PAGE_SIZE,
+  TITLE_FOR_NON_ADMIN_ACTION,
+} from '../../constants/constants';
+import { Status, Webhook } from '../../generated/entity/events/webhook';
+import { useAuth } from '../../hooks/authHooks';
+import { Button } from '../buttons/Button/Button';
+import ErrorPlaceHolder from '../common/error-with-placeholder/ErrorPlaceHolder';
+import NextPrevious from '../common/next-previous/NextPrevious';
+import NonAdminAction from '../common/non-admin-action/NonAdminAction';
+import WebhookDataCard from '../common/webhook-data-card/WebhookDataCard';
+import { leftPanelAntCardStyle } from '../containers/PageLayout';
+import { WebhooksProps } from './Webhooks.interface';
+import './webhookV1.less';
+
+const statuses = [
+  {
+    label: startCase(Status.Disabled),
+    value: Status.Disabled,
+  },
+  {
+    label: startCase(Status.Active),
+    value: Status.Active,
+  },
+  {
+    label: startCase(Status.Failed),
+    value: Status.Failed,
+  },
+  {
+    label: startCase(Status.AwaitingRetry),
+    value: Status.AwaitingRetry,
+  },
+  {
+    label: startCase(Status.RetryLimitReached),
+    value: Status.RetryLimitReached,
+  },
+];
+
+const WebhooksV1: FC<WebhooksProps> = ({
+  data = [],
+  paging,
+  selectedStatus = [],
+  onAddWebhook,
+  onClickWebhook,
+  onPageChange,
+  onStatusFilter,
+  currentPage,
+}) => {
+  const { isAuthDisabled, isAdminUser } = useAuth();
+  const [filteredData, setFilteredData] = useState<Array<Webhook>>(data);
+
+  const getFilteredWebhooks = () => {
+    return selectedStatus.length
+      ? data.filter(
+          (item) => item.status && selectedStatus.includes(item.status)
+        )
+      : data;
+  };
+
+  const fetchRightPanel = () => {
+    return (
+      <Card data-testid="data-summary-container" style={leftPanelAntCardStyle}>
+        <div className="tw-my-2">
+          The webhook allows external services to be notified of the metadata
+          change events happening in your organization through APIs. Register
+          callback URLs with webhook integration to receive metadata event
+          notifications. You can add, list, update, and delete webhooks.
+        </div>
+      </Card>
+    );
+  };
```

Comment:
```suggestion
  const rightPanel = useMemo(() => {
    return (
      <Card data-testid="data-summary-container" style={leftPanelAntCardStyle}>
        <div className="tw-my-2">
          The webhook allows external services to be notified of the metadata
          change events happening in your organization through APIs. Register
          callback URLs with webhook integration to receive metadata event
          notifications. You can add, list, update, and delete webhooks.
        </div>
      </Card>
    );
  }, []);
```

Let's avoid unwanted re-renders for this functions.